pyfarm/models/jobqueue.py

```diff
@@ -196,7 +197,12 @@ def get_job_for_agent(self, agent):
                         if (object.can_use_more_agents() and
                             object.num_assigned_agents() + 1 <
                                 (object.maximum_agents or maxsize)):
-                            return object
+                            if PREFER_RUNNING_JOBS:
+                                return object
```

Comment:
I just realized this but `object` needs to be renamed, it shadows the builtin object.